src/test/java/de/retest/recheck/ui/diff/AttributeDifferenceTest.java

```diff
@@ -2,56 +2,39 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
-import org.junit.jupiter.api.Test;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.SystemOutRule;
 
-import de.retest.recheck.ui.diff.AttributeDifference;
+public class AttributeDifferenceTest {
 
-class AttributeDifferenceTest {
+	@Rule
+	public final SystemOutRule systemOutRule = new SystemOutRule().enableLog();
```

Comment:
Is there a Junit5 way to do this? You would have to use the correct Test annotations.